Mage.Sets/src/mage/cards/i/IrreverentGremlin.java

```diff
@@ -0,0 +1,60 @@
+package mage.cards.i;
+
+import java.util.UUID;
+
+import mage.MageInt;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.costs.common.DiscardCardCost;
+import mage.abilities.effects.common.DoIfCostPaid;
+import mage.abilities.effects.common.DrawCardSourceControllerEffect;
+import mage.abilities.keyword.MenaceAbility;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.SubType;
+import mage.filter.common.FilterCreaturePermanent;
+import mage.filter.predicate.mageobject.AnotherPredicate;
+import mage.filter.predicate.mageobject.PowerPredicate;
+
+/**
+ * @author paasar
+ */
+public final class IrreverentGremlin extends CardImpl {
+    private static final FilterCreaturePermanent creatureWithPower2OrLessFilter = new FilterCreaturePermanent("creature with power 2 or less");
```

Comment:
text is `another creature you control with power 2 or less`

Reply:
Excellent observations, everyone!

I did some changes and it should be fixed now. Please have an another look, when you have the time.